diff --git a/bubblewrap.c b/bubblewrap.c
index e05f697..504512a 100644
--- a/bubblewrap.c
+++ b/bubblewrap.c
@@ -791,13 +791,6 @@ acquire_privs (void)
       /* Historically we supported this, but now we only do user namespaces */
       die ("setuid use of bubblewrap is not supported");
     }
-  else if (real_uid != 0 && has_caps ())
-    {
-      /* We have some capabilities in the non-setuid case, which should not happen.
-         Probably caused by the binary being setcap instead of setuid which we
-         don't support anymore */
-      die ("Unexpected capabilities but not setuid, old file caps config?");
-    }
   else if (real_uid == 0)
     {
       /* If our uid is 0, default to inheriting all caps; the caller
